tests/integration/tier1/destructive/excluded-packages-removed/test_excluded_pkgs_removed.py

```diff
@@ -34,4 +34,4 @@ def test_excluded_packages_removed(shell, convert2rhel):
     assert c2r.exitstatus == 0
 
     # Verify, the excluded packages were really removed
-    assert shell(f"rpm -qi {packages}").returncode != 0
+    assert shell(f"rpm -qi {packages}").returncode == 1
```

Comment:
```suggestion
    assert shell(f"rpm -qi {packages}").returncode != 0
```
This needs to be reverted, since rpm returns the code based on a number of "failed" packages.
So if we query for just one, and it is indeed not installed `returncode == 1`
If we query for more packages at once, which we do, the returncode would be equal to number of said packages.
```
>>> print(subprocess.run(["rpm", "-q", "kernel*", "kernel-core*"]))
package kernel* is not installed
package kernel-core* is not installed
CompletedProcess(args=['rpm', '-q', 'kernel*', 'kernel-core*'], returncode=2)

>>> print(subprocess.run(["rpm", "-q", "k", "e", "r", "n", "e", "l"]))
package k is not installed
package e is not installed
package r is not installed
package n is not installed
package e is not installed
package l is not installed
CompletedProcess(args=['rpm', '-q', 'k', 'e', 'r', 'n', 'e', 'l'], returncode=6)
```
Actually it might even be something like 
```suggestion
    pkgs_count = len(packages.split(" "))
    assert shell(f"rpm -qi {packages}").returncode == pkgs_count
```

Reply:
Oh, I did not know about that rpm return code functionality. I like the approach with the `len(packages.split(" "))`